server.py

```diff
@@ -48,16 +49,13 @@ def start(self):
         print("[STARTING] server is starting...")
         self.server.listen(2)
         print(f"[LISTENING] server is listening on {self.SERVER}")
-        thread = threading.Thread(target=self.start_game_when_two_clients)
-        thread.start()
+        threading.Thread(target=self.start_game_when_two_clients).start()
 
-        i = 1
         while True:
             conn, addr = self.server.accept()
             clients.append((conn, addr))
 
-            print(f"[ACTIVE CONNECTIONS] {i}")
-            i += 1
+            print(f"[ACTIVE CONNECTIONS] {len(clients)}")
```

Comment:
- Count active connections with number of clients, no longer number of times loop has run